Backpack/src/main/java/net/skyscanner/backpack/button/BpkButtonLink.kt

```diff
@@ -53,16 +55,22 @@ open class BpkButtonLink @JvmOverloads constructor(
     }
 
   init {
-    defaultTextColor = ContextCompat.getColor(this.context, R.color.bpkSkyBlue)
+    var textColor = ContextCompat.getColor(context, R.color.bpkPrimary)
 
     compoundDrawablePadding = tokens.bpkSpacingSm
 
     this.context.theme.obtainStyledAttributes(attrs, R.styleable.BpkButtonLink, defStyleAttr, 0)
-      ?.use {
+      .use {
         _uppercase = it.getBoolean(R.styleable.BpkButtonLink_uppercase, true)
+        textColor = it.getColor(R.styleable.BpkButton_buttonTextColor, textColor)
       }
 
     background = resolveThemeDrawable(context, android.R.attr.selectableItemBackground)
+    setTextColor(getColorSelector(
+      textColor,
+      darken(textColor, .1f),
```

Comment:
Why is darken always used now? It was only for the disabled variation before

Reply:
If you take a look in the master, each time you press the button link the colour becomes darker.